### ISSUE TITLE
No graphs for tunnel interfaces with if_ plugin over munin-async

### ISSUE THREAD

**Issue by brknkfr** [munin-monitoring/munin#1188]
**Describe the bug**
No graphs generated for tunnel (openvpn) interfaces like tun-u1194 or tun-t1194.

**To Reproduce**
I installed a openvpn server on a debian stretch system with two tunnel interfaces (tun-u1194 and tun-t1194, one for tcp and one for udp). With an up to date munin version from backports (version 2.0.47-1~bpo9+1) no graphs are generated on the munin master for the tunnel interfaces.

Running `sudo -u munin munin-run if_tun-u1194` works as expected. Other graphs are working aswell. The node runs with munin-async. There are no warnings or errors in the munin-update.log

```
sudo -u munin munin-run if_tun-u1194
# Warning: Root privileges are required to change user/group.  The plugin may not behave as expected.
down.value 151120
up.value 653236
```

**Comment by brknkfr** [comment]
Correction: It works, when connecting directly to port 4949 and not over munin-async! Apparently, there has to be a problem with the if_ alias interfaces and munin-async.

**Comment by sumpfralle** [comment]
Thank you for your report!

This issue (see #951) was resolved only a few weeks ago in munin v2.0.48.

The backports package for 2.0.49 should be available quite soon, I guess.
